src/main/java/ch/njol/skript/expressions/ExprIndicesOfValue.java

```diff
@@ -107,113 +110,112 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 
 		position = matchedPattern <= 1;
 		string = matchedPattern == 0;
-		value = LiteralUtils.defendExpression(exprs[0]);
-		objects = exprs[1];
+		needle = LiteralUtils.defendExpression(exprs[0]);
+		haystack = exprs[1];
 
-		return LiteralUtils.canInitSafely(value);
+		return LiteralUtils.canInitSafely(needle);
 	}
 
 	@Override
 	protected Object @Nullable [] get(Event event) {
-		Object value = this.value.getSingle(event);
-		if (value == null)
-			return (Object[]) Array.newInstance(getReturnType(), 0);
+		Object[] needle = this.needle.getArray(event);
```

Comment:
what about `positions of 1, 2, or 3 in {_list::*}`? I would expect that to work like AND, so maybe use getAll here.